Makefile

```diff
@@ -200,6 +204,14 @@ install:
 	$(INSTALL) -m 0755 -d $(DESTDIR)$(BINDIR)
 	$(INSTALL) -m 0755 ./tetragon $(DESTDIR)$(BINDIR)
 
+.PHONY: install-operator 
+install-operator:
```

Comment:
This rule install both Tetragon agent and operator in a Kind cluster, so `kind-install-tetragon` would be better IMO. Also, it should depend on `kind` and `build`/`images`.